Cargo.toml

```diff
@@ -19,7 +19,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 yansi = "0.5"
 fs-err = "2.9"
-target-lexicon = { version = "0.12", features = ["serde_support"] }
+target-lexicon = { version = "0.12.11", features = ["serde_support"] }
```

Comment:
You didn't actually upgrade by changing the version from 0.12 to 0.12.11, you forced cargo to download that specific version.

Leave the version as 0.12 and run an `cargo update` instead. This will update all the dependencies to the latest version possible. I.e. if 0.12 is specified, it will look for the latest 0.12.* version. This is why I always specify deps with major.minor so that all patch version are easy to update.